catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java

```diff
@@ -323,7 +333,8 @@ private void addMetacardFormat(final XstreamPathValueTracker pathValueTracker,
             MetacardImpl metacard) {
         String format = pathValueTracker.getFirstValue(toPath(GmdMetacardType.FORMAT_PATH));
         if (StringUtils.isNotEmpty((format))) {
-            metacard.setAttribute(GmdMetacardType.GMD_FORMAT, format);
+            // TODO is media.format better? Dublin core makes media.format seem more like physical format
```

Comment:
I believe `media.format` is more appropriate here; `GmdMetacardType.FORMAT_PATH` expresses the file format (ex. GeoTiff)